### DISCUSSION TITLE
How to add a variation to one of base game's songs using `_merge` and/or `_append` logic to make it playable in freeplay?

### DISCUSSION THREAD

**Discussion by MAZ12211** [FunkinCrew/Funkin#3800] [Q&A]
I have a High remix I want to chart and make it playable as a variation called `duet` instead of making it a new song/entry in freeplay.
[I followed the docs on how to do this](https://funkincrew.github.io/funkin-modding-docs/02-custom-songs-and-custom-levels/02-05-adding-variations-to-existing-songs.html#if-the-song-is-from-the-base-game-or-a-different-mod) but nothing happened in freeplay, so I'm still having trouble
`v0.5.3`

Here's my folder tree:
```
mymod/
├─ _merge/
│  ├─ songs/
│  │  ├─ high/
│  │  │  ├─ Inst-duet.ogg
│  ├─ data/
│  │  ├─ songs/
│  │  │  ├─ high/
│  │  │  │  ├─ high-duet-chart.json
│  │  │  │  ├─ high-duet-metadata.json
│  │  │  │  ├─ high-metadata.json
```
high-metadata.json has
```
[
    { "op": "add", "path": "/playData/songVariations/-", "value": "duet" }
]
```
[Here's the tool I used for making that tree folder](https://ascii-tree-generator.com/) if somebody knows where I'm meant to place those files

**Comment by AbnormalPoof** [comment]
I'm pretty sure it's because you are merging to `high-duet-chart.json` and `high-duet-metadata.json`, I'm assuming you want to add those files. Try moving them to `data/songs/high/` instead of `_merge/data/songs/high/`

**Reply by MAZ12211** [reply]
Same result, the variation doesn't exist in freeplay when trying to switch difficulties for high (only default and erect works)
I tried moving `Inst-duet.ogg` out of `_merge` (so it's _only_ `high-metadata.json` in the `_merge` folder) but it's the same result

Here's the mod file if it's needed
[random-pack.zip](https://github.com/user-attachments/files/17558287/random-pack.zip)

**Comment by MAZ12211** [comment]
Still confused on this lol
